### ISSUE TITLE
Different loss calculations

### ISSUE THREAD

**Issue by shimazadeh** [shimazadeh/Neural_Networks#2]
Right now the model uses binary cross entropy to calculate loss, can we add other options as well?

**Comment by shimazadeh** [comment]
we have different methods of calculating loss and each one is suited for a specific problem:
1. MSE: for regression problems
2. Binary cross entropy: for classification problem
3. Cross entropy: for multi-class classification problem
4. SVM loss: this is used for support vector machines and some neural nets
5. Huber loss: combines MSE and MAE 

the neural network in this project uses Binary cross entropy formula since our Y is a binary classification (either a cancerous or non-cancerous tissue). that is why we use Binary cross entropy BUT if someone wants to use our model for some other type of problems can they use it?
